Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -2660,8 +2669,413 @@ def fetch_incidents():
+            if last_scan_time == first_fetch_timestamp:
+                # This means that this is the first fetch, and we want to include resources with a scan time
+                # EQUAL or GREATER than the first fetch timestamp
+                filter = f"{filter} >='{last_scan_time}'"
+            else:
+                # This means that this is not the first fetch, and we only want to include resources with a scan time
+                # GREATER than the last configured scan time, to prevent duplicates.
+                filter = f"{filter} >'{last_scan_time}'"
+        fetch_limit = current_fetch_info_detections.get('limit') or INCIDENTS_PER_FETCH
+        # Default filter for first fetch
+        fetch_query = demisto.params().get('iom_fetch_query')
+        validate_iom_fetch_query(iom_fetch_query=fetch_query)
+        if fetch_query and not iom_next_token:
+            # If the user entered a fetch query, then append it to the filter
+            filter = f"{filter}+{fetch_query}"
+        demisto.debug(f'IOM {filter=}')
+        iom_resource_ids, iom_new_next_token = iom_ids_pagination(filter=filter,
+                                                                  iom_next_token=iom_next_token,
+                                                                  fetch_limit=fetch_limit,
+                                                                  api_limit=500)
+        demisto.debug(f'Fetched the following IOM resource IDS: {", ".join(iom_resource_ids)}')
+        # Since the first API call is only in charge of retrieving the IDs of the iom resources, we need
+        # another API call to actually get the entity/detail of each resource.
+        iom_resources = get_iom_resources_for_fetch(iom_resource_ids=iom_resource_ids)
+        for iom_resource in iom_resources:
+            resource_id = iom_resource.get('id', '')
+            if resource_id not in last_fetch_resource_ids:
+                demisto.debug(f'Creating an incident for CrowdStrike IOM with ID: {resource_id}')
+                iom_resource['incident_type'] = 'iom_configurations'
+                iom_resource_to_context = create_incident_from_iom_resource(iom_resource=iom_resource)
+                iom_config_incidents.append(iom_resource_to_context)
+                # The time_scan date returned from the API contains nanoseconds, which is yet to be
+                # supported when using datetime.strptime(some_time, format), therefore, we need to convert the
+                # date to the supported format (IOM_DATE_FORMAT).
+                scan_time = format_time_to_custom_date_format(iom_resource.get('scan_time', ''), IOM_DATE_FORMAT)
+                # Other than creating an incident out of the iom resources, we also need to save the latest
+                # scan time
+                if datetime.strptime(scan_time, IOM_DATE_FORMAT) > datetime.strptime(new_scan_time, IOM_DATE_FORMAT):
+                    new_scan_time = scan_time
```

Comment:
may be easier/prettier to just keep them in a list, then select max(...), rather than parse&compare over and over